README.md

```diff
@@ -228,20 +228,22 @@ Check out the Secrets section in the following READMEs:
 
 #### Test all or some packages
 
+- Run `yarn start infrastructure` before running tests to start services.
```

Comment:
If you're actively developing and you initially ran `yarn start` to start everything up, there's a good chance these infra services _are_ already running. Might be worth updating this to say something like "You might need to run..." 🤷

Reply:
Ohh yeah. It's only required when we aren't developing anything. I will add as you said.

Reply:
Done